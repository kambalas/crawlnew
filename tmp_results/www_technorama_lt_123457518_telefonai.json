[
  {
    "name": "Telefonas Apple iPhone 15 Pro Max 256GB Natural Titanium",
    "price": "",
    "error": false
  },
  {
    "name": "Telefonas Samsung Galaxy S24 Ultra 5G 12GB/512GB Titanium Black",
    "price": "",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 16 Pro Max 256GB Black Titanium",
    "price": "",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 128GB Black",
    "price": "",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 16 Pro 128GB Desert Titanium",
    "price": "",
    "error": false
  }
]