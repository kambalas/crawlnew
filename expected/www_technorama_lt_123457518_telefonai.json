[
  {
    "name": "iPhone 15 Pro Max",
    "price": "1499",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 Pro Max 256GB Natural Titanium",
    "price": "1959490485318",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 16 Pro Max 256GB Black Titanium",
    "price": "1959498056916",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 15 128GB Black",
    "price": "1959490359824",
    "error": false
  },
  {
    "name": "Telefonas Apple iPhone 16 Pro 128GB Desert Titanium",
    "price": "1959497714602",
    "error": false
  }
]